### PR TITLE
Add support Intel NUC

### PR DESCRIPTION
**Pull request by salerio-gs**
This adds support for running SEAPATH on Intel NUC. This is done by adding a new MACHINE (votp-nuc) and some additions to the kernel configuration to support MMC, NVMe and Intel LPSS.

### CODE REVIEW

File: recipes-kernel/linux/linux-mainline-rt/defconfig [hunk 1]
```diff
@@ -279,3 +279,17 @@ CONFIG_PROVIDE_OHCI1394_DMA_INIT=y
...
+CONFIG_MMC_BLOCK=y
+CONFIG_MMC_BLOCK_MINORS=8
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_PCI=y
+CONFIG_MMC_RICOH_MMC=y
+CONFIG_MMC_SDHCI_ACPI=y
+CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_NVME_CORE=y
+CONFIG_BLK_DEV_NVME=y
+
```
Comment (ebail) [inline recipes-kernel/linux/linux-mainline-rt/defconfig:295]: Great to see that in defconfig as it is not related to NUC.
Could you please commit that in a separate commit message.

File: conf/machine/votp.conf [hunk 2]
```diff
@@ -6,7 +6,6 @@
 #@SOC:
 #@DESCRIPTION: Machine configuration based on x86-64 processor
 
-MACHINE = "votp"
```
Comment (ebail) [inline conf/machine/votp.conf:9]: Not sure it is right. We will test it.

**Review by ebail — CHANGES_REQUESTED** [review]
Looks good, except committing unrelated changes to NUC in a separate commit message.
I would not use uppercase in your commit title: 

 conf/machine/votp-nuc.conf: add support Intel NUC

File: recipes-kernel/linux/linux-mainline-rt/defconfig (hunk 1 above)
Comment (salerio-gs) [reply recipes-kernel/linux/linux-mainline-rt/defconfig:295]: Yes

File: conf/machine/votp.conf (hunk 2 above)
Comment (salerio-gs) [reply conf/machine/votp.conf:9]: I have never seen a MACHINE statement in a yocto MACHINE definition file. What is certain is if you leave it in there you can't add a separate machine definition which includes the original i.e. votp.conf. If you do that the build fails.

**Comment by salerio-gs** [comment]
I will update...

File: conf/machine/votp.conf (hunk 2 above)
Comment (dupremathieu) [reply conf/machine/votp.conf:9]: I confirm we can remove "Machine" safely.

**Review by dupremathieu — APPROVED** [review]
I managed to generate both images (votp & votp-nuc) and I didn't see any regression.

**Pull request merged**